cordapp-cpk2/src/main/kotlin/net/corda/plugins/cpk2/VerifyBundle.kt

```diff
@@ -107,6 +108,11 @@ open class VerifyBundle @Inject constructor(objects: ObjectFactory) : DefaultTas
         }
     }
 
+    private fun verifyExportPackage(verifier: Verifier) = verifier.exportPackage.entries
+        .map { it.key }
+        .filter { it.startsWith("net.corda.") }
```

Comment:
What happens if someone puts something into the `net.corda` package itself?

Reply:
It did fail. I've added a test to cover it and updated to code to detect `net.corda` using a regex.

I also added a test to check `net.cordapp` is allowed.